deployment/gem.rb

```diff
@@ -108,6 +108,19 @@ def ask(question, default)
   end
 end
 
+namespace :sumodev do
+  namespace :db do
+    desc "Create the database if it doesn't exists yet"
+    task :create do
+      sumo.db.create
```

Comment:
Why would they be deprecated? Our other deploy scripts use these sumodev style commands, so I would not remove them in the short future.